File: src/main/java/br/com/jiratorio/service/EstimateService.java
package br.com.jiratorio.service;

import br.com.jiratorio.domain.form.EstimateForm;
import br.com.jiratorio.domain.vo.EstimateIssue;

import java.util.List;

public interface EstimateService {

    List<EstimateIssue> findEstimateIssues(Long boardId, EstimateForm estimateForm);
}

File: src/main/kotlin/br/com/jiratorio/service/EstimateService.kt
package br.com.jiratorio.service

import br.com.jiratorio.domain.request.SearchEstimateRequest
import br.com.jiratorio.domain.response.EstimateIssueResponse

interface EstimateService {

    fun findEstimateIssues(boardId: Long, searchEstimateRequest: SearchEstimateRequest): List<EstimateIssueResponse>

}
